.github/workflows/_build_publish.yaml

```diff
@@ -133,6 +133,7 @@ jobs:
           asset_prefix: image_${{ matrix.image }}-amd64
           image: ./build/docker/${{ matrix.image }}-amd64.tar
           upload-sbom-release-assets: true
+          skip_cis_scan: true
```

Comment:
Hi, what's the different between this property and just putting `if: false` onto the step?
I'll approve this change ftm, but if `if: false` applies, I'd rather use it.